rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(book, person, date)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] When a rental is created it should add itself to the `rentals` of the book and person objects. That way the relationship between the book and the person through the rental is established.

  ```
  def initialize(book, person, date)
    @date = date
    @book = book
    @person = person

    book.rentals << self
    person.rentals << self
  end
  ```

Reply:
Resolved